addons/web/static/src/webclient/webclient.scss

```diff
@@ -76,6 +76,12 @@ div.o_boolean_toggle.custom-control.custom-checkbox {
   }
 }
 
+// User input typically entered via keyboard
+kbd {
+  border: 1px solid $o-gray-200;
+  box-shadow: 0px 1px 1px rgba($black, 0.2), inset 0px -1px 1px 1px rgba($o-gray-200, 0.8), inset 0px 2px 0px 0px rgba($white, 0.8);
```

Comment:
Looks like there is a `$kbd-box-shadow` Bootstrap variable, maybe check if customizing it gives the desired result?

Reply:
I have customized the `$kbd-box-shadow` variable.
But the `box-shadow` property in `webclient.scss` is still needed because shadows are disabled in Enterprise : https://github.com/odoo/enterprise/blob/1b613d0409a425261d69c00ab98d639f85f72b9b/web_enterprise/static/src/legacy/scss/bootstrap_overridden.scss#L18

Reply:
Naive question but... is there (stil?) a reason for disabling shadows in Enterprise?

Asking because I see a lot of place where we add (manually) shadows anyway...

Ping @stefanorigano

Reply:
> Naive question but... is there (stil?) a reason for disabling shadows in Enterprise?

Bootstraps adds tons of box-shadows for decorative purposes (eg. buttons' `:focus` states...) ... generally I prefer to define the right shadow for each component that needs it, rather than override in raw SCSS all the unwanted shadows for each component/state.

I didn't check yet if BTS5 introduces new tools for fine-tune the default behaviour, but it's eventually something we can check later.